go/arm/Dockerfile.tmpl

```diff
@@ -3,11 +3,16 @@ ARG VERSION
 ARG TAG_EXTENSION=''
 FROM --platform=linux/amd64 ${REPOSITORY}/golang-crossbuild:${VERSION}-base${TAG_EXTENSION} as stage-amd64
 
+# Needed by libsystemd-dev 247.3-7+deb11u4
+{{- if eq .DEBIAN_VERSION "11"}}
+RUN echo "deb http://deb.debian.org/debian bullseye-updates main" >> /etc/apt/sources.list
+{{- end }}
+
```

Comment:
The list of sources has a file per Debian version, To add the line to the source here is not the correct way in this repo, see https://github.com/elastic/golang-crossbuild/blob/main/go/base/sources-debian11.list